src/pretix/locale/uk/LC_MESSAGES/django.po

```diff
@@ -3,97 +3,105 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
+#: pretix/plugins/stripe/apps.py:14
 #, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2024-06-22 12:20+0000\n"
+"POT-Creation-Date: 2024-06-23 10:10+0000\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"Last-Translator: AbhishekG <gawandeabhi123@gmail.com>\n"
+"Language-Team: Ukrainian <https://translate.pretix.eu/projects/pretix/pretix/uk/>\n"
+
 "Language: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+"Plural-Forms: nplurals=4; plural=(n % 1 == 0 && n % 10 == 1 && n % 100 != "
```

Comment:
How did you generate this code? The error in [Docker build job](https://github.com/fossasia/eventyay-tickets/actions/runs/9624347608/job/26547954453#step:4:3318) is related to this.


> ValueError: invalid token in plural form: EXPRESSION

Reply:
Oh..., maybe the error message is caused by other file, because in this hunk, you are deleting `EXPRESSION`. Gettext will not see it.

Reply:
this django.po file is generated using = python manage.py makemessages -l uk --->> this files extracts the translation 
and then = python manage.py compilemessages to compile the messages.